multi-tool instructions (#376)
nf-core/modules: README.md

    > NB: Build information for all tools within a multi-tool container can be obtained in the `/usr/local/conda-meta/history` file within the container.

- It is also possible for a new multi-tool container to be built and added to BioContainers by submitting a pull request on their [`multi-package-containers`](https://github.com/BioContainers/multi-package-containers) repository.
    - Fork the [multi-package-containers repository](https://github.com/BioContainers/multi-package-containers)

>     - Make a change to the hash.tsv file in the combinations directory see [here](https://github.com/aunderwo/multi-package-containers/blob/master/combinations/hash.tsv#L124) for an example where I added pysam=0.16.0.1,biopython=1.78

Reviewer: ```suggestion
    - Make a change to the `hash.tsv` file in the `combinations` directory see [here](https://github.com/aunderwo/multi-package-containers/blob/master/combinations/hash.tsv#L124) for an example where I added `pysam=0.16.0.1,biopython=1.78`
```

>     - Make a change to the `hash.tsv` file in the `combinations` directory see [here](https://github.com/aunderwo/multi-package-containers/blob/master/combinations/hash.tsv#L124) for an example where I added `pysam=0.16.0.1,biopython=1.78`

Reviewer: ```suggestion
    - Make a change to the `hash.tsv` file in the `combinations` directory see [here](https://github.com/aunderwo/multi-package-containers/blob/master/combinations/hash.tsv#L124) for an example where `pysam=0.16.0.1,biopython=1.78` was added.
```

>     - Commit the code and then make a pull request to the original repo for [example](https://github.com/BioContainers/multi-package-containers/pull/1661)

Reviewer: ```suggestion
    - Commit the code and then make a pull request to the original repo, for [example](https://github.com/BioContainers/multi-package-containers/pull/1661)
```

    - Once the PR has been accepted a container will get built and you can find it using  a search tool in the `galaxy-tool-util conda` package

>       ```bash

Reviewer 2: Here you could use `console` instead of `bash`

Reviewer: Nice! Thanks @rivera10 ! You made those changes already right?

      mulled-search --destination quay singularity conda  --search pysam biopython  | grep "mulled"
      quay         mulled-v2-3a59640f3fe1ed11819984087d31d68600200c3f  185a25ca79923df85b58f42deb48f5ac4481e91f-0  docker pull quay.io/biocontainers/mulled-v2-3a59640f3fe1ed11819984087d31d68600200c3f:185a25ca79923df85b58f42deb48f5ac4481e91f-0
      singularity  mulled-v2-3a59640f3fe1ed11819984087d31d68600200c3f  185a25ca79923df85b58f42deb48f5ac4481e91f-0  wget https://depot.galaxyproject.org/singularity/mulled-v2-3a59640f3fe1ed11819984087d31d68600200c3f:185a25ca79923df85b58f42deb48f5ac4481e91f-0
      ```

>     - These can be used to add to the relevant docker and singularity lines in the PROCESS definition

Reviewer: ```suggestion
    - You can copy and paste the `mulled-*` path into the relevant Docker and Singularity lines in the Nextflow `process` definition of your module
```

    - To confirm that this is correct. Spin up a temporary Docker container

>       ```bash

Reviewer 2: Here too

      docker run --rm -it quay.io/biocontainers/mulled-v2-3a59640f3fe1ed11819984087d31d68600200c3f:185a25ca79923df85b58f42deb48f5ac4481e91f-0  /bin/sh
      ```

      And in the command prompt type

      ```console

>       / # grep specs /usr/local/conda-meta/history

Reviewer: ```suggestion
      $ grep specs /usr/local/conda-meta/history
```
